rct229/rules/section22/section22rule19.py

```diff
@@ -0,0 +1,74 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.utils.jsonpath_utils import find_all, find_exactly_one_with_field_value
+
+APPLICABLE_SYS_TYPES = [
+    "SYS-7",
+    "SYS-8",
+    "SYS-11.1",
+    "SYS-11.2",
+    "SYS-12",
+    "SYS-13",
+    "SYS-7B",
+    "SYS-8B",
+    "SYS-11B",
+    "SYS-12B",
+    "SYS-13B",
+]
+REQUIRED_TEMP_RESET = schema_enums["TemperatureResetOptions"]
```

Comment:
I suggest using the name of the enum without the "Options" at the end.  So in this case it would be `TEMPERATURE_RESET`

Reply:
I will constantly use (schema_enums_name without options). Thanks for this Charlie.